addons/hw_drivers/tools/helpers.py

```diff
@@ -262,12 +272,18 @@ def get_ssid():
     process_grep = subprocess.Popen(['grep', 'ESSID:"'], stdin=process_iwconfig.stdout, stdout=subprocess.PIPE)
     return subprocess.check_output(['sed', 's/.*"\\(.*\\)"/\\1/'], stdin=process_grep.stdout).decode('utf-8').rstrip()
 
+
 def get_odoo_server_url():
     if platform.system() == 'Linux':
         ap = subprocess.call(['systemctl', 'is-active', '--quiet', 'hostapd']) # if service is active return 0 else inactive
         if not ap:
             return False
-    return read_file_first_line('odoo-remote-server.conf')
+
+    if (odoo_server := read_file_first_line('/home/pi/odoo-remote-server.conf')) is None:
+        _logger.error('Failed to read odoo-remote-server.conf')
```

Comment:
Since by default the IoT Box isn't connected to a server this gives us error logs in a loop, it should be avoided